internal/cmd/internal/olmv1/operator_update.go

```diff
@@ -0,0 +1,40 @@
+package olmv1
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	v1action "github.com/operator-framework/kubectl-operator/internal/pkg/v1/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+// NewOperatorUpdateCmd allows updating a selected operator
+func NewOperatorUpdateCmd(cfg *action.Configuration) *cobra.Command {
+	i := v1action.NewOperatorUpdate(cfg)
+	i.Logf = log.Printf
+
+	cmd := &cobra.Command{
+		Use:   "operator <operator>",
+		Short: "Update an operator",
```

Comment:
do we need a `Long` description here as well? I noticed some olmv0 commands have pretty comprehensive `Long` descriptions. 

I was thinking something similar (but shorter) to what kubectl explain provides for our CRDs could be useful?

Reply:
> do we need a Long description here as well? 

+1 to long description

Reply:
> For OLM v1, we should move away from `operator` terminology. We should call it clusterextension. @joelanford WDYT?

IMO while the `operator olmv1` command prefix (olmv1 need to start with `kubectl operator olmv1` prefix) is still in, I think we should probably stick with using the `operator` terminology (olmv0 also did not work with `operator` CRDs), but if people think it's important to switch asap, I don't mind.

BTW. we had(?) this discussion internally on slack - maybe we shouldn't duplicate here?

Reply:
> BTW. we had(?) this discussion internally on slack - maybe we shouldn't duplicate here?

I do not remember of any discussion we had on this. Also the intention is not to duplicate it.

Reply:
This is the link - https://redhat-internal.slack.com/archives/C08DV3H57L2/p1740592186952269. Looks like you took part